line-bot-servlet/src/main/java/com/linecorp/bot/servlet/LineBotCallbackRequestParser.java

```diff
@@ -39,9 +39,10 @@
     private final ObjectMapper objectMapper;
     private final LineBotClient lineBotClient;
 
-    public LineBotCallbackRequestParser(LineBotClient lineBotClient, ObjectMapper objectMapper) {
```

Comment:
The following code will fail because it doesn't set `FAIL_ON_UNKNOWN_PROPERTIES` `false`.
```
new LineBotCallbackRequestParser(client, new ObjectMapper());
```
I think It's confusing. 

And I'm not sure the reason for passing `ObjectMapper` instance in this case. I suggest removing the parameter from constructor.